src/core_plugins/kibana/public/dev_tools/partials/dev_tools_app.html

```diff
@@ -1,15 +1,24 @@
 <div class="dev-tools__app-container app-container">
-  <bread-crumbs class="bread-crumbs--navbar" omit-current-page="true"></bread-crumbs>
   <kbn-top-nav name="devtools" config="kbnDevToolsApp.topNavConfig">
-    <div class="localTabs">
-      <a
-        ng-repeat="item in kbnDevToolsApp.devTools"
-        class="localTab"
-        ng-class="{'localTab-isSelected': kbnDevToolsApp.currentPath === item.url}"
-        kbn-href="{{::item.url}}"
-      >
-        {{::item.display}}
-      </a>
+    <!-- Transcluded elements. -->
+    <div data-transclude-slots>
+      <!-- Breadcrumbs. -->
+      <bread-crumbs
+        data-transclude-slot="topLeftCorner"
+        omit-current-page="true"
+      ></bread-crumbs>
+
+      <!-- Tabs. -->
+      <div data-transclude-slot="bottomRow" class="localTabs">
+        <a
+          ng-repeat="item in kbnDevToolsApp.devTools"
+          class="localTab"
+          ng-class="{'localTab-isSelected': kbnDevToolsApp.currentPath === item.url}"
```

Comment:
I need to fix the UI Framework to use this `localTab-isSelected` format. Currently, it's `localTab-is-selected`.